### ISSUE TITLE
Is it possible to pin the table header so it doesn't scroll out of view?

### ISSUE THREAD

**Issue by mattnedrich** [facebook/fixed-data-table#470]

**Comment by KamranAsif** [comment]
Isn't that what the header already does?

**Comment by jayongg** [comment]
No, it does not. Do you have an example app showing the table header doesn't scroll out of view?  None of the examples here https://facebook.github.io/fixed-data-table/example-column-groups.html show any scrolling.
